crates/compiler/can/src/def.rs

```diff
@@ -1181,35 +1219,32 @@ fn canonicalize_value_defs<'a>(
             }
             PendingValue::InvalidIngestedFile => { /* skip */ }
             PendingValue::ImportNameConflict => { /* skip */ }
+            PendingValue::StmtAfterExpr => { /* skip */ }
         }
     }
 
     let mut symbol_to_index: Vec<(IdentId, u32)> = Vec::with_capacity(pending_value_defs.len());
 
     for (def_index, pending_def) in pending_value_defs.iter().enumerate() {
-        let mut new_bindings = BindingsFromPattern::new(pending_def.loc_pattern()).peekable();
-
-        if new_bindings.peek().is_none() {
-            env.problem(Problem::NoIdentifiersIntroduced(
-                pending_def.loc_pattern().region,
-            ));
-        }
+        if let Some(loc_pattern) = pending_def.loc_pattern() {
```

Comment:
for your consideration: it seems that [let-else](https://doc.rust-lang.org/beta/rust-by-example/flow_control/let_else.html) should work for `continue` as well, I think that may keep this from getting more indented.

Reply:
ah right, nice!